### PR TITLE
TPROD-363: Deleting unused php templates

### PR DESCRIPTION
**Pull request by escopecz**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

These bundles were already refactored from PHP templates to Twig but some PHP templates were forgotten or the use of them. This PR cleans it up.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. This affects many bundles. Some changes touched these features:
3. Form submit asset download action - The message should be formatted based on the theme used in the instance or form
4. SMS send action should be displayed in the contact detail's timeline

### CODE REVIEW

File: app/bundles/UserBundle/Controller/RoleController.php [hunk 1]
```diff
@@ -279,7 +279,7 @@ public function editAction($objectId, $ignorePost = true)
 
         return $this->delegateView([
             'viewParameters' => [
-                'form'              => $form->createView(), //$this->setFormTheme($form, 'MauticUserBundle:Role:form.html.php', 'MauticUserBundle:FormTheme\Role'),
+                'form'              => $form->createView(),
```
Comment (JoshuaEstes) [inline app/bundles/UserBundle/Controller/RoleController.php:282]: For these I would make sure the forms still render correctly. Are the themes being used?

**Review by mabumusa1 — APPROVED** [review]
I did a code review, and did not notice critical issues with the files.

**Comment by sdoering** [comment]
After having flagged a user as DNC (email) I (as a sales user) can't remove thw DNC flag by clicking the little "x" in the contact view page at the DNC marker (orange bar). It results in an Error 500: 
image

Entry from the log file: 
```
[2023-03-01 13:32:58] mautic.CRITICAL: Uncaught PHP Exception TypeError: "Mautic\CoreBundle\Factory\ModelFactory::getModel(): Return value must be of type Mautic\CoreBundle\Model\MauticModelInterface, Mautic\LeadBundle\Model\DoNotContact returned" at /var/www/html/app/bundles/CoreBundle/Factory/ModelFactory.php line 43 {"exception":"[object] (TypeError(code: 0): Mautic\\CoreBundle\\Factory\\ModelFactory::getModel(): Return value must be of type Mautic\\CoreBundle\\Model\\MauticModelInterface, Mautic\\LeadBundle\\Model\\DoNotContact returned at /var/www/html/app/bundles/CoreBundle/Factory/ModelFactory.php:43)"} {"hostname":"mautic-web","pid":1998}
```

**Comment by escopecz** [comment]
@sdoering I created https://github.com/mautic/mautic/issues/12053 for your find. It's not caused by this PR but needs fixing.

File: app/bundles/UserBundle/Controller/RoleController.php (hunk 1 above)
Comment (escopecz) [reply app/bundles/UserBundle/Controller/RoleController.php:282]: I believe this is handled by https://github.com/mautic/mautic/pull/12046. I'm lacking focus to the community in the past 2 weeks but don't want to block the merges.

**Pull request merged**